TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java

```diff
@@ -299,6 +345,9 @@ protected static SslContext loadBacksideSslContext(URI serverUri, boolean allowI
     }
 
     protected static Map<String, String> convertPairListToMap(List<String> list) {
+        if (list == null) {
+            return Map.of();
+        }
         var map = new TreeMap<String, String>();
```

Comment:
Should we use a `LinkedHashMap` to preserve ordering?

Reply:
Sure